qt/python/mantidqt/widgets/codeeditor/multifileinterpreter.py

```diff
@@ -69,16 +69,21 @@ def _tab_title_and_tooltip(self, filename):
         if filename is None:
             title = NEW_TAB_TITLE
             i = 1
-            while title in self.tab_titles:
+            while title in self.stripped_tab_titles:
                 title = "{} ({})".format(NEW_TAB_TITLE, i)
                 i += 1
             return title, title
         else:
             return osp.basename(filename), filename
 
     @property
-    def tab_titles(self):
-        return [self._tabs.tabText(i).rstrip('*') for i in range(self.editor_count)]
+    def stripped_tab_titles(self):
+        tab_text = [self._tabs.tabText(i) for i in range(self.editor_count)]
+        tab_text = [txt.rstrip('*') for txt in tab_text]
+        # Some IDEs (such as KDE) will automatically assign keyboard shortcuts using the Qt & annotation
```

Comment:
I, personally, find IDE to be slightly confusing language, maybe specifying Desktop Environment would be better?

Reply:
DE was the correct word I was looking for, force of habit typing IDE
Thanks for catching that